src/variants.ini

```diff
@@ -1018,6 +1018,16 @@ flagRegionBlack = *1
 flagMove = true
 checking = false
 
+# Dodo Chess
+# Win by campmate. No checks allowed.
+# Variant defined in Pychess discord and previously playable with Chessboi bot.
+[dodo:chess]
```

Comment:
I do not have a strong opinion on whether to add dodo as a built-in or a configured variant, but I would prefer to have it only as one of the two (whichever you prefer) to avoid redundancy.